### PR TITLE
Resetting the screen height to auto.

### PR DESCRIPTION
**Pull request by Tristan3D3**
Resetting the screen height to auto because it caused some funny glitches. #22 & #122 

This should hopefully fix it :)

### CODE REVIEW

**Comment by awtkns** [comment]
@Tristan3D3 would you be able to share before and after images?

File: src/styles/globals.css [hunk 1]
```diff
@@ -75,3 +75,8 @@ div::-webkit-scrollbar-thumb {
 div::-webkit-scrollbar-button {
   display: none;
 }
+
+/* Height of page was limited so changing it back to auto */
+.h-screen {
```
Comment (awtkns) [inline src/styles/globals.css:80]: ideally we want do be doing these changes only in tailwind.

Comment (Tristan3D3) [reply src/styles/globals.css:80]: > @Tristan3D3 would you be able to share before and after images?

Before :
image

After :
image

**Comment by Tristan3D3** [comment]
> @Tristan3D3 would you be able to share before and after images?

Before :
image

After :
image

**Comment by Charlz4500** [comment]
[Apr]()

File: src/styles/globals.css (hunk 1 above)
Comment (Tristan3D3) [reply src/styles/globals.css:80]: This just changes the "h-screen" property in tailwind to default to auto if not set
I think it's kinda still tailwind

**Comment by awtkns** [comment]
@Tristan3D3 doesnt seem to be working for me :( 

Screen Shot 2023-04-15 at 4 55 34 PM

**Comment by rahul-ghimire-au6** [comment]
hey @awtkns, to re-create this issue please turn on chrome's dev mode, by "right-clicking" then clicking "inspect"
please select responsive and add this screen resolutions 1336 * 600 
have added support fix for this issue in this pr and also screen shots (https://github.com/reworkd/AgentGPT/pull/158)

**Comment by Tristan3D3** [comment]
> @Tristan3D3 doesnt seem to be working for me :(
> 
> Screen Shot 2023-04-15 at 4 55 34 PM

Okay I see
Currently working on a fix

File: src/styles/globals.css (hunk 1 above)
Comment (Tristan3D3) [reply src/styles/globals.css:80]: > ideally we want do be doing these changes only in tailwind.

PR #159 resolves this :)

**Comment by Tristan3D3** [comment]
> Okay I see Currently working on a fix

Issue fixed in PR #159 :)

**Pull request closed without merging**